tsc/scripts/git_count.py

```diff
@@ -0,0 +1,495 @@
+#!/usr/bin/env python3
```

Comment:
What if a specific user has quit an organization? Their github ID will no more have details of the previous organization and the final org counts will not be correct. Anyway to overcome this problem?